### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by Gegardus**
In this part of the "Math magicians" project were writing unit tests with React Testing Library and snapshots with Jest.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,14 @@ Feel free to check the [issues page](https://github.com/Gegardus/math-magicians/
 - GitHub: [@githubhandle](https://github.com/Gegardus)
 - Twitter: [@twitterhandle](https://twitter.com/Gegardus)
```
Comment (Meltrust) [inline README.md:49]: - [x] Please, kindly add the "testing" section from the [Microverse readme template](https://github.com/microverseinc/readme-template) to your project, and include instructions on how to run the tests.

That way, you will be able to showcase your hard work to potential employers and users 💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @Gegardus,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- Nice code organization✔️ 
- Nice usage of testing framework✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Gegardus) [reply README.md:49]: Done. Thank you @Meltrust.

**Review by Meltrust — APPROVED** [review]
Hi @Gegardus,


## Wow, you did it 🎉
Brilliant
## Thank you for adding the "Run tests" section to your readme. 💪 🥇 ㊗️ 
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Gegardus** [comment]
Thank you @Meltrust Have a great day!

**Pull request merged**